lolex.js

```diff
@@ -172,12 +172,10 @@ function withGlobal(_global) {
     }
 
     function createDate() {
-        function ClockDate(year, month, date, hour, minute, second, ms) {
+        function createNativeDate(year, month, date, hour, minute, second, ms) {
```

Comment:
Revert all changes to `lolex.js`. This is changed only by build scripts.

Reply:
Done